csm-1.4/MTL-2423-xfs-opts/README.md

```diff
@@ -0,0 +1,41 @@
+# CSM 1.4 XFS Mount Options
+
+This hotfix fixes lengthy IMS jobs that occur after upgrading to COS 2.5 that can be attributed to the XFS mount options present on NCN Kubernetes nodes.
+
+The hotfix will
+
+- Munge all XFS mount points defined in `/etc/fstab.metal`with the fixed mount options
+- Deploy a new `csm-config` for including a newer `dracut-metal-mdsquash` RPM in image builds that contain the mount options
+- Updates cray-product-catalog
+- Updates CFS configurations for NCNs
+- Builds new NCN images
+
+## JIRAs
+
+This hotfix covers the following JIRAs:
+
+* [MTL-2423](https://jira-pro.it.hpe.com:8443/browse/MTL-2423)
```

Comment:
Technically this hotfix also contains the changes for CASMINST-6896: Support for multiple GPG keys. It may be worth mentioning, since we have a separate hotfix out there for just that, and you wouldn't want to apply them separately if you want both of them.

Reply:
This hotfix only contains _some_ of those changes, it contains a newer `csm-config` but it does not contain the bits necessary for applying the new keys.

That being said, this hotfix needs to be ran _after_ the other hotfix, if they are ran in a different order then CASMINST-6896 will undo this hotfix by rolling csm-config backwards.

It's a bit tedious to have the hotfixes separate, but I don't imagine we'll get the approval to merge the two together. I also just completed testing, and as much as I want to make these better I am not keen on signing up for another go. :(

I'll update the directions here to require the other hotfix.